script/release.sh

```diff
@@ -71,6 +71,7 @@ clone() {
   cd $CLI
   git remote remove origin
   git remote add origin git@github.com:$ORIGIN_REPO/$CLI.git
+  git remote add upstream git@github.com:$UPSTREAM_REPO/$CLI.git
```

Comment:
But you never push to upstream.

This is why I suggested https, so auth prompt pops-up and you know that you are pushing to wrong remote.